lib/importproject.cpp

```diff
@@ -232,111 +232,165 @@ static std::string readUntil(const std::string &command, std::string::size_type
     return ret;
 }
 
+static std::string removeBackslash(std::string str)
+{
+    str.erase(std::remove(str.begin(), str.end(), '\\'), str.end());    
+    return str;
+}
+
+static void skip_whitespaces(const std::string& command, std::string::size_type *pos)
+{
+    while (*pos < command.size() && command[*pos] == ' ')
+        (*pos)++;
+}
+
+
+static std::string parseTillNextCommandOpt(const std::string& command, std::string::size_type *pos)
+{
+    std::string ret{};
+    while (*pos < command.size() && command[*pos] != '/' && command[*pos] != '-' )
```

Comment:
hmm.. not your code. But as far as I see this loop can be written as:

    *pos = command.find_first_not_of("/-", *pos);

Reply:
good catch, I will change it